### DISCUSSION TITLE
News Cache [2023-10-09]

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#4425] [Announcements]
### 🚢  Ships

- [Comments now available in Vercel's Slack integration](https://vercel.com/changelog/comments-now-available-in-vercels-slack-integration)
- [Track server-side custom events with Vercel Web Analytics](https://vercel.com/changelog/track-server-side-custom-events-with-vercel-web-analytics)
- [Spend Management now available for Pro users](https://vercel.com/changelog/spend-management-now-available-for-pro-users)
    - [Introducing Spend Management](https://vercel.com/blog/introducing-spend-management-realtime-usage-alerts-sms-notifications)

### 💡 Inspiration

- [Jennifer Fisher's online store was built with Next 13 and Shopify, and deployed with Vercel](https://x.com/seif_ghezala/status/1711475025348583687?s=20)

### 📚 Education

- Watch the [Vercel AI Accelerator playlist](https://www.youtube.com/playlist?list=PLBnKlKpPeagmocQSY50TUMfpSj9QLGFU0) on YouTube
- [Get involved with SvelteKit during Hacktoberfest](https://svelte.dev/blog/hacktoberfest)

**Comment by Josephtran102** [comment]
I cant verify my phone number. I sent DM you on X bt not reply. No one support for Vercel???

**Reply by amyegan** [reply]
Sorry I missed your message on X! You should be able to log in now without your phone number. Please let me know if you still have trouble with your account.
